source/api/image/3.0/index.md

```diff
@@ -435,11 +435,13 @@ In order to support the above requirements, clients _SHOULD_ construct image req
 | --------- | --------------- |
 | region    | "full" if the full image is requested<br/>otherwise the `x,y,w,h` syntax. |
 | size      | "max" if the maximum size is requested,<br/>otherwise the `w,h` syntax. |
-| rotation  | "!" if the image is mirrored, followed by an integer if possible, and trimming any trailing zeros in a decimal value, and a leading 0 if the value is below 1. |
+| rotation  | "!" if the image is mirrored, followed by an integer if possible, otherwise a floating point value. |
 | quality   | "default" if the server's default quality is requested,<br/>otherwise the quality string. |
 | format    | An explicit format string is always required. |
 {: .api-table}
 
+Size and region parameters given as percentages and the rotation parameter allow positive floating point number values. Integer values _SHOULD_ be use where possible. When floating point values are used, they _SHOULD_ consist only of decimal digits and "." (e.g. 0.9 not +0.9), be represented with a leading 0 if less than 1 (e.g. 0.9 not .9), and not include trailing zeros (e.g. 0.9 not 0.90). Intermediate calculations may use floating point numbers and the rounding method is implementation specific.
```

Comment:
> _SHOULD_ consist only of decimal digits and a "."

Why not MUST?

Reply:
We could do MUST for the first clause (decimal digits and "."), however then we'd probably want to relax to SHOULD for the other clauses. I did consider this but don't feel strongly. Interested in other opinions